namespace cpp entity
namespace py ccentity.gem
namespace as3 com.dreamfire.sanguo.data.definition.entity

////////////////////////////////////////////////
// 宝石位置类型定义
enum GemLocationType {
  MIN = 0,

  GEM_STORAGE = 0,                    // 宝石仓库
  GEM_EQUIP,                          // 宝石装备

  MAX,
}

